src/client/utility-types.ts

```diff
@@ -40,6 +40,12 @@ export function* utilityTypes(): DocumentParts {
   )
   yield `}`
   yield `export type AppClientComposeExecuteParams = Pick<SendTransactionParams, 'skipWaiting' | 'maxRoundsToWaitForConfirmation' | 'populateAppCallResources' | 'suppressLog'>`
+  yield NewLine
+  yield `export type IncludeSchema = {`
+  yield IncIndent
+  yield* jsDoc(`The storage schema to request for the created app`)
+  yield `schema?: AppStorageSchema`
```

Comment:
Don't merge until https://github.com/algorandfoundation/algokit-utils-ts/pull/250 has been merged, and package version has been bumped here